testutil/validatormock/propose.go

```diff
@@ -138,11 +149,11 @@ func ProposeBlock(ctx context.Context, eth2Cl eth2wrap.Client, signFunc SignFunc
 			Message:   block.Capella,
 			Signature: sig,
 		}
-	default:
+	default: // TODO(xenowits): Add a case for deneb block.
```

Comment:
why this is not added in this PR as you have already added case for deneb almost everywhere?

Reply:
yeah intended to, and I then got lazy. I have added it now for both `ProposeBlock` & `ProposeBlindedBlock`